subprojects/android-test/ui-testing-core/src/main/kotlin/com/avito/android/test/util/Instrumentations.kt

```diff
@@ -7,6 +7,10 @@ import androidx.test.runner.lifecycle.ActivityLifecycleMonitorRegistry
 import androidx.test.runner.lifecycle.Stage
 
 fun getCurrentActivity(): Activity {
+    return getCurrentActivitySafe() ?: throw IllegalStateException("Resumed activity not found")
+}
+
+fun getCurrentActivitySafe(): Activity? {
```

Comment:
```suggestion
fun getCurrentActivitySafely(): Activity? {
```
OR
```suggestion
fun getCurrentActivityOrNull(): Activity? {
```
`OrNull` is a commonly used naming pattern in such cases.

Reply:
Changed to OrNull